.github/CODEOWNERS

```diff
@@ -1832,6 +1832,8 @@ spec/support/vcr_cassettes/lighthouse/auth/client_credentials/invalid_client_id_
 spec/support/vcr_cassettes/lighthouse/auth/client_credentials/invalid_scopes_400.yml @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/vfs-facilities
 spec/support/vcr_cassettes/lighthouse/auth/client_credentials/revoke_consent_204.yml @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/vfs-facilities
 spec/support/vcr_cassettes/lighthouse/auth/client_credentials/token_200.yml @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/vfs-facilities
+spec/support/vcr_cassettes/lighthouse/benefits_claims/documents/lighthouse_document_upload_200_jpg.yml @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+spec/support/vcr_cassettes/lighthouse/benefits_claims/documents/lighthouse_document_upload_200_pdf.yml @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
will this be required for all new files added to vets-api? seems like there should be some higher level rules rather then file by file

Reply:
You can add one line for `spec/support/vcr_cassettes/lighthouse/benefits_claims/documents/`, or even `spec/support/vcr_cassettes/lighthouse/benefits_claims/` if all files in that dir would have the same owners.

Reply:
oh, good to know. thanks, updated to cover the directory.